README.md

```diff
@@ -1,6 +1,6 @@
 # Usage
 ```
 bash setup.sh
-java -jar tictactoe-server-0.1.0-standalone.jar [PORT=5000]
+java -jar tictactoe-server-0.1.0-standalone.jar [-p PORT=5000]
```

Comment:
The README probably needs more detail about where this jar comes from, and what Java version is required - I got it running but needed more information than what's here to do it.